### ISSUE TITLE
Missing fields for complete Chorus import

### ISSUE THREAD

**Issue by brazzmonkey** [akretion/factur-x-libreoffice-extension#5]
Hi there,
Thanks for this extension and for ODS template file.
There seems to be a few missing fields when importing generated PDF in Chorus pro :
- VAT type
- Payment mode
- contract number
- and, ideally: bank account detail
Is there any plans to add those fields in ODS template and XML output?
Thanks

*edit*
Basically, what I'm asking is for the script to use BASIC WL profile instead of MINIMUM.

**Comment by brazzmonkey** [comment]
It looks like in 2024 MINIMUM profile will not be accepted in France, BASIC or BASIC WL profile should be considered for this extension.

**Comment by alexis-via** [comment]
My opinon is that a spreadsheet is too "open" to allow reliable generation of profils higher than MINIMUM. If you want higher profils, I think you should opt for an opensource ERP such as Odoo for example.
But it's not very difficult to modify to implement higher profils. You need a very strict spreadsheet organisation to handle multiple VAT rates.

**Comment by brazzmonkey** [comment]
I agree, handling multiple VAT rates requires more complex tools, but in my opinion that's not the point here.
It seems to that an extension such as yours is targeted to very small businesses, who need to generate simple, basic invoices. Therefore I suppose handling only one VAT rate should be enough for most of them (if not, indeed they probably already use an ERP or any invoicing software that supports multiple VAT rates).

In addition, providing only MINIMUM profile in your extension will inevitably make it obsolete.
I wish I could help coding those simple improvements, but I have no coding skills for LO macros.

**Comment by alexis-via** [comment]
AFAIK, Chorus Pro still accepts the MINIMUM profile, and Chorus Pro is the current target.
I'm fully aware that the e-invoicing reform for France will be a game changer... but it's starting on September 2026 (any it's not post-poned a second time !). In the meantime, the MINIMUM profile should work perfectly well for Chorus Pro, which is the main use-case for small companies at the moment.
